<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarMainWindow</class>
 <widget class="QMainWindow" name="CalendarMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>248</width>
      <height>155</height>
     </rect>
    </property>
    <property name="firstDayOfWeek">
     <enum>Qt::Monday</enum>
    </property>
    <property name="gridVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoTodayButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>I dag</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contactlistButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kontaktliste</string>
    </property>
   </widget>
   <widget class="QLabel" name="chosenDateLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valgt dato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addAppointmentButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Legg til ny avtale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeAppointmentButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Slett avtale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editAppointmentButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Endre avtale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeAllAppointmentsButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Slett alle avtaler</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="appointmentTable">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="typeLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="locationLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sted</string>
    </property>
   </widget>
   <widget class="QLabel" name="contactLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Kontakt</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="typeLineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="locationLineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="contactLineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>350</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="infoLineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>111</width>
      <height>64</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lagre og lukk</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
